### PR TITLE
Add update on class fields

### PR DESCRIPTION
**Pull request by mathiasbynens**
**Target Live Date:** asap

- [x] This has been reviewed and approved by the team
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2018/12/class-fields.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+counter.value;
+// logs 'Getting the current value!'
+// → 0
+counter.increment();
+counter.value;
+// logs 'Getting the current value!'
+// → 1
+```
+
+Note that accessing the `value` executes some code (i.e., it logs a message) before returning the
```
Comment (petele) [inline src/content/en/updates/2018/12/class-fields.md]: Did you want to use the `Note:` syntax for the blue box?

File: src/content/en/updates/2018/12/class-fields.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+```
+
+## Conclusion
+
+Public class fields are shipping in V8 v7.2 and Chrome 72. We plan on shipping private class fields
+soon.
+
+Questions about this new feature? Comments about this article? Feel free to ping me on Twitter via
+[@mathias](https://twitter.com/mathias)!
+
```
Comment (petele) [inline src/content/en/updates/2018/12/class-fields.md]: Please add the was this helpful widget: `{% include "web/_shared/helpful.html" %}`

File: src/content/en/updates/2018/12/class-fields.md (hunk 1 above)
Comment (mathiasbynens) [reply src/content/en/updates/2018/12/class-fields.md]: Not in this case. It’s part of the general flow, and doesn’t need a separate callout.

File: src/content/en/updates/2018/12/class-fields.md (hunk 2 above)
Comment (mathiasbynens) [reply src/content/en/updates/2018/12/class-fields.md]: Done!

**Pull request merged**